### ISSUE TITLE
definitions.h error expected identifier

### ISSUE THREAD

**Issue by brendangregg** [iovisor/bpftrace#164]
```
# ./src/bpftrace -e 'tracepoint:syscalls:sys_exit_s* { @[name] = count(); }'
definitions.h:3:1: error: expected identifier or '('
Attaching 69 probes...
^C

@[tracepoint:syscalls:sys_exit_select]: 32
```

It's an unnecessary error, and can be removed.

Maybe related to #163 /CC @danobi . Debug output:

```
bpftrace -d -e 'tracepoint:syscalls:sys_exit_sched_geta* { @[pid] = count(); }'
#include <linux/types.h>
struct _tracepoint_syscalls_sys_exit_sched_geta*
{
};

Program
 tracepoint:syscalls:sys_exit_sched_geta*
  =
   map: @
    builtin: pid
   call: count

definitions.h:3:1: error: expected identifier or '('
[...]
```

Note the empty struct.

Once that error is removed, there's another thing to improve:

```
bpftrace -e 'tracepoint:syscalls:sys_exit_s* { @[args->ret] = count(); }'
definitions.h:3:1: error: expected identifier or '('
Struct/union of type '_tracepoint_syscalls_sys_exit_s*' does not contain a field named 'ret'
```

For now, we might want that last error to say "args is not yet supported with probe wildcards". Later on we should investigate making it work (it'd have to check that args->XXX is valid for all expanded probes, and probably needs to set need_expansion too).

**Comment by danobi** [comment]
Yeah I think I broke some stuff with the other patch. I'll go fix that.

**Comment by danobi** [comment]
I think I can try to take on the other bits as well.

**Comment by danobi** [comment]
#165 should fix the empty structs.

**Comment by brendangregg** [comment]
fixed by #165
